middleware/proxy.go

```diff
@@ -108,15 +108,15 @@ func proxyRaw(t *ProxyTarget, c echo.Context) http.Handler {
 			return
 		}
 
-		errc := make(chan error, 2)
+		errChan := make(chan error, 2)
```

Comment:
Please revert the naming to original for consistency.

Reply:
isn't it more readable and clear?

Reply:
Ok, let's make it `errCh`.

Reply:
ok

Reply:
fixed